core-cube/src/main/java/org/apache/kylin/cube/CubeManager.java

```diff
@@ -394,6 +411,14 @@ private void setCubeMember(CubeInstance cube, CubeUpdate update) {
                 cube.putSnapshotResPath(lookupSnapshotPathEntry.getKey(), lookupSnapshotPathEntry.getValue());
             }
         }
+
+        if (update.getCreateTimeUTC() >= 0) {
+            cube.setCreateTimeUTC(update.getCreateTimeUTC());
```

Comment:
Same as above, why to reset the "createTimeUTC" on an update, that is a little strange.